tests/ui/generic-associated-types/issue-90014-tait.rs

```diff
@@ -13,7 +13,6 @@ struct Foo<'a>(&'a mut ());
 
 impl Foo<'_> {
     type Fut<'a> = impl Future<Output = ()>;
-    //^ ERROR: the type `&mut ()` does not fulfill the required lifetime
```

Comment:
Are you sure that removing these is the right call? Whoever added them probably intended the error to be shown...

Reply:
This test is marked as a known-bug, which doesn't allow error annotations anyways